src/ScottPlot5/ScottPlot/Pixel.cs

```diff
@@ -20,4 +20,14 @@ public override string ToString()
     {
         return $"Pixel: X={X}, Y={Y}";
     }
+
+    public static Pixel operator +(Pixel a, Pixel b)
```

Comment:
Nice! This is exactly the spirit of these new primitive types, and I'm excited to build functionality like this out as ScottPlot 5 matures.

After merging this PR I'll move this to Common and add regression tests and XML docs. Let's not do that in this PR, because it's already complex enough as it is.